net/picotls/Portfile

```diff
@@ -21,6 +21,11 @@ long_description    {*}${description}. &{name} is designed to be fast, tiny and
                     with the primary user being the H2O HTTP/2 server for serving \
                     HTTP/1, HTTP/2 and HTTP/3 over QUIC.
 
+set with_fusion     OFF
+if {${build_arch} in [list i386 x86_64]} {
```

Comment:
Sorry for nitpicking, but `configure.build_arch` should rather be used, IMO, here (though it makes difference only for edge cases).